README.md

```diff
@@ -17,41 +17,53 @@
 </p>
 
 
-Transformers Interpret is a model explainability tool designed to work exclusively with 🤗 transformers.
+Transformers Interpret is a model explainability tool designed to work exclusively with the [transformers][transformers] package.
 
-In line with the philosophy of the transformers package tranformers interpret allows any transformers model to be explained in just two lines. It even supports visualizations in both notebooks and as savable html files.
+In line with the philosophy of the transformers package Tranformers Interpret allows any transformers model to be explained in just two lines. It even supports visualizations in both notebooks and as savable html files.
 
-This package stands on the shoulders of the the incredible work being done by the teams at [Pytorch Captum](https://captum.ai/) and [Hugging Face](https://huggingface.co/) and would not exist if not for the amazing job they are both doing in the fields of nlp and model interpretability respectively.
 
 ## Install
 
-I recommend Python 3.6 or higher, Pytorch 1.5.0 or higher, transformers v3.0.0 or higher, and captum 0.3.1 (**required**). The package does not work with Python 2.7.
```

Comment:
Broken down is more readable. Also, aren't all of these required?

Reply:
Yeah I think it broken down is better. So the only explicit requirement is captum 0.3.1. It does work with prior versions of pytorch and transformers but I haven't tested it with those so want to recommend a base level of compatibility that I can guarantee.

Reply:
Right, you mean that the only requirement for that specific version, don't you? Because for instance `Pytorch 1.5.0 or higher` is also a requirement, isn't it?

Reply:
More so that I have tested out on versions of Pytorch > 1.5.0 and amn't sure about lower version, they might work, they also might not so I thought I'd make them strong recommendations. Captum 0.3.1 is a definite requirement though.

Reply:
Right gotcha! I think the way I'd phrase it in this case is that your 'supported' versions are as as follows, instead of 'required'. Saying 'Supported: Pytorch > 1.5.0, Captum == 0.3.1' is clear enough I'd say.

Reply:
Agreed, that's much more clear.

Reply:
cool 👍